### PR TITLE
Add github action to check release

### PR DESCRIPTION
**Pull request by hbcarlos**

### CODE REVIEW

File: .github/workflows/check-release.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+name: Check Release
+on:
+  push:
+    branches:
+      - main
```
Comment (jtpio) [inline .github/workflows/check-release.yml:5]: The repo still has `master` as the default branch.

We could actually do the rename to `main`. Then update the other workflow here: https://github.com/QuantStack/jupyterlab-js-logs/blob/e081f732312db1f733c0d55672b037cffe217a80/.github/workflows/build.yml#L5

**Comment by jtpio** [comment]
As a heads-up: the releaser now supports a list of PyPI tokens, that can be saved as `PYPI_TOKEN_MAP` in the GitHub secrets: https://github.com/jupyter-server/jupyter_releaser#checklist-for-adoption

**Comment by jtpio** [comment]
Kicking CI

**Comment by jtpio** [comment]
The check release is looking good:

image

**Comment by hbcarlos** [comment]
I'm looking at your PR in jupyterlab-link-share. What else is missing here for the new version of Jupiter-releaser?

**Comment by jtpio** [comment]
I think this should be good to go since the check release workflow looks good and is passing.

If you want to do the release, the next steps would be:

- set up the `PYPI_TOKEN_MAP` as mentioned here: https://github.com/jupyter-server/jupyter_releaser#checklist-for-adoption
- Run the Draft Changelog workflow from the releaser fork
- Merge the changelog PR here in this repo
- Run the Full Publish workflow from the releaser fork

**Comment by hbcarlos** [comment]
Thanks!! I'll do it tomorrow morning.

**Review by jtpio — APPROVED** [review]
Thanks!

**Pull request merged**